x-pack/plugins/security_solution/public/app/home/home_navigations.ts

```diff
@@ -120,13 +119,6 @@ export const navTabs: SecurityNav = {
     disabled: false,
     urlKey: 'cases',
   },
-  [SecurityPageName.administration]: {
```

Comment:
The removal of this tab config, is related to: https://github.com/elastic/kibana/pull/137541#discussion_r932693620

and this updated line of code: 
```
  const mapLocationToTab = useCallback(
    (): string => (navTabs[pageName] || (tabName && navTabs[tabName]))?.id ?? '', 
[pageName, tabName, navTabs])
```
By default within security solution we are able to match against pageName which is the intended way to find what tab show as active. However the management (administration) team opted to use the tabName instead. This causes conflicts with route matching, and is problematic if tab on any page has the same name as a tab on the sidebar.

To force mapLocationToTab to handle the unique case of administrations method. we removed that key from here, forcing the default case `navTabs[pageName]` to fail, and allowing the admin tabs to be matched based on their tabName